erpnext/shopping_cart/cart.py

```diff
@@ -86,6 +85,29 @@ def place_order():
 				if item.qty > item_stock.stock_qty[0][0]:
 					throw(_("Only {0} in Stock for item {1}").format(item_stock.stock_qty[0][0], item.item_code))
 
+	#if checkout without payment has been enabled, submit the quotation, and convert to sales order
+	if cart_settings.sales_team_order_without_payment:
+	
+		quotation.requested_delivery_date = getdate(delivery_date)
+		quotation.save(ignore_permissions=True)
+		quotation.submit()
+		frappe.db.commit()
+
+		from erpnext.selling.doctype.quotation.quotation import _make_sales_order
+		sales_order = frappe.get_doc(_make_sales_order(quotation.name, ignore_permissions=True))
+		sales_order.payment_schedule = []
+		sales_order.transaction_date = nowdate()
+		sales_order.delivery_date = getdate(delivery_date)
```

Comment:
You should fix the get_mapped_doc inside Quotation rather than setting the value here

https://github.com/Bloomstack/erpnext/blob/production/erpnext/selling/doctype/quotation/quotation.py#L127